timestamp/README.md

```diff
@@ -0,0 +1,50 @@
+# Timestamp Modules
```

Comment:
Could you create a new commit for this README.md file?
It is contained in " Force single thread for GraphQL tokio runtime " commit.

Reply:
OMG. My mistake. I'll remove it